README.md

```diff
@@ -1 +1,126 @@
-# catalog_of_my_things
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Catalog] <a name="about-project"></a>
+
+**[Catalog]** Allows users to register new music albums, books and games and to preserve their data.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>Back-end</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Preserves data locally]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+ruby installed 
+
+### Setup
+
+Clone this repository to your desired folder:
+
+  git clone git@github.com:Juanpa8830/catalog_of_my_things.git
+  cd catalog_of_my_things
+
+
+### Install
+
+Install this project with:
+
+  bundle install
+
+### Usage
+
+To run the project, execute the following command:
+
+ruby app.rb
```

Comment:
## [Optional]
Please correct the file path because your main file is main.rb which is placed inside the classes folder. Kindly mention it here as well 👍 
```
cd classes
ruby main.rb
```